app/components/markdown/markdown_table_row/index.tsx

```diff
@@ -45,9 +45,11 @@ const MarkdownTableRow = ({isFirstRow, isLastRow, children}: MarkdownTableRowPro
     // Add an extra prop to the last cell so that it knows not to render a right border since the container
     // will handle that
     const renderChildren = React.Children.toArray(children) as ReactElement[];
-    renderChildren[renderChildren.length - 1] = React.cloneElement(renderChildren[renderChildren.length - 1], {
-        isLastCell: true,
-    });
+    if (renderChildren.length > 0) {
```

Comment:
This is needed after these changes since the limit could possibly produce a table row without children which wasn't possible before